inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/enums/FieldType.java

```diff
@@ -31,6 +31,7 @@ public enum FieldType {
     FLOAT,
     DECIMAL,
     STRING,
+    TEXT,
```

Comment:
I think that there is no need to define a text here, there are already general types such as string/varchar that can be expressed.

Reply:
Postgres source connector may meet TEXT field type.